basic-javascript/project-1.js

```diff
@@ -31,23 +37,26 @@ const greaterThanFifty = num => {
 };
 
 const add = (x, y) => {
-  return x + y;
+  return ~~x + ~~y;
```

Comment:
Interesting! I had not seen this before. From Stack Overflow, seems like a faster substitute for Math.floor?